src/modules/review/dto/create.review.dto.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  IsNotEmpty,
+  IsNumber,
+  IsString,
+  IsUUID,
+  Max,
+  Min,
+} from 'class-validator';
+import { ApiProperty } from '@nestjs/swagger';
+
+export class CreateReviewDto {
+  @ApiProperty({
+    description: 'Content of the review.',
+    example: 'Great accommodation, would stay again!',
+  })
+  @IsNotEmpty({ message: 'Content is required.' })
+  @IsString()
+  content: string;
```

Comment:
Please change ```@IsNotEmpty()``` to ```@IsOptional()``` decorator and make the type optional too, as ```content``` is not required for a rating